coriolis_openstack_utils/actions/tenant_actions.py

```diff
@@ -203,6 +203,14 @@ def execute_operations(self):
 
         return new_project_id
 
+    def cleanup(self):
+        for action in self.subactions:
+            action.cleanup()
+
+        self._destination_openstack_client.delete_project_by_name(
+            self.get_new_tenant_name())
+        return {'done': True}
```

Comment:
Why do you return this here? If anything, it should be `{'done': False}` because it undone, no?